### ISSUE TITLE
Nav Bar Button fix

### ISSUE THREAD

**Issue by imanishbarnwal** [hack4bengal/hack4bengal.github.io#61]
**Describe the bug**
When you go to the events page or team page you will still see the home page button is highlighted instead of the page I am on

**Expected behavior**
When I press the events page or go directly to the events page/ team page, the events option should be highlighted instead of the Home button.

**Screenshots**

`Events Page`

![image](https://user-images.githubusercontent.com/46371923/194472036-17f11f51-a95c-40cc-93b1-e8e466d6fde1.png)

**Comment by Parth-Panjwani** [comment]
Hi, I want to contribute to this issue.

**Comment by imanishbarnwal** [comment]
Sure go ahead @Parth-Panjwani

**Comment by MohitSinghChauhan** [comment]
Assign it to me also @imanishbarnwal

**Comment by Parth-Panjwani** [comment]
Yes @imanishbarnwal I have almost completed working on it. It will be my first pull request 🥳 Thank You for assigning me to it😇

**Comment by Parth-Panjwani** [comment]
Hi @imanishbarnwal , I have fixed the issue and have requested for pull request. :)
